views/page-not-found.html

```diff
@@ -1,4 +1,14 @@
-{% extends "layout.html" %}
+{% if isRegistration %}
```

Comment:
There is a blocked ticket in K1 to page not found: https://github.com/companieshouse/overseas-entities-web/pull/904 I'm happy for this change here.